package.json

```diff
@@ -225,6 +225,12 @@
         "build:icons"
       ]
     },
+    "new": {
+      "command": "./tools/create-element/bin/main.js",
+      "dependencies": [
+        "build:create"
```

Comment:
Looks like `npm run new` is hanging after the `What is the element's name?` message

Reply:
This fix definitely works, we couldn't figure out why the chaining `&&` was working but the `dependencies` in wireit command weren't working in a similar fashion.  Nice fix brian.